### ISSUE TITLE
Question about LR

### ISSUE THREAD

**Issue by mingqiJ** [ayushjain1144/odin#24]
Hi, I see in all sh files, you just use the base_lr to train. You used default setting of Detectron. 

Could you tell me how to decrease the learning rate in the sh file. I see you said something in the sh file like : reduce lr at 32k iterations to 1e-5, get the best checkpoint at 1.5k iterations

**Comment by ayushjain1144** [comment]
I think you would need to manually change `SOLVER.BASE_LR 1e-4` to `SOLVER.BASE_LR 1e-5` in the sh script. 

You can also set `SOLVER.STEPS: (32000)` and it would automatically decrease lr to 1/10th, but I would be careful about it -- the convergence steps depend a lot on effective batch size etc, so I find it safer to reduce the learning rate when the model converges at the base learning rate

**Comment by mingqiJ** [comment]
I see. Thank you. I am curious about the learning rate you used, then I can directly follow your setting.

**Comment by ayushjain1144** [comment]
Sure. I used lr of 1e-4 for all experiments to start with. And then decreased to 1e-5 when the model converged on 1e-4. The exact number of steps after which I decreased are different for different datasets, and I listed them as comments in each of the bash script.
